src/test/java/Utils/CommandTest.java

```diff
@@ -9,7 +9,7 @@
 
 public class CommandTest {
 
-	Command command = new Command("help", (argv) -> {
+	Command command = new Command("help","", (argv) -> {
```

Comment:
This change does not do much, as the constructor should work without help string too

Reply:
This was just for fun